doc/Custom.md

```diff
@@ -1,3 +1,6 @@
+### URL
+Conduktor Platform is available on [http://localhost:8080](http://localhost:8080)
+
 ### Connect to an unsecured kafka cluster accessible on `0.0.0.0:9092`
```

Comment:
Given we provide MacOS details below, I feel we should now amend this one back to Linux (with --network host) param instead. Also update the heading `Local Kafka on Linux`, WDYT?